index.html

```diff
@@ -155,19 +158,38 @@ <h2>
       </h2>
       <section data-link-for="Navigator">
         <h2>
-          Permissions Policy Integration
+          Permissions Integration
         </h2>
         <p>
-          {{requestMIDIAccess()}} is a <a>policy-controlled feature</a>, as
-          defined by [[[Permissions-Policy]]].
-        </p>
-        <p>
-          The feature name for {{requestMIDIAccess()}} is <dfn data-lt=
-          "midi-fp" data-lt-nodefault="">`midi`</dfn>.
+          The Web Midi API is a [=default powerful feature=] that is identified
```

Comment:
Just a "powerful feature": 

```suggestion
          The Web Midi API is a [=powerful feature=] that is identified
```

Default ones don't define additional things 😅

Reply:
Whoops. 🙈